addons/binding/org.openhab.binding.russound/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="russound"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="rio">
+        <label>Russound RIO Device</label>
+        <description>Ethernet access point to Russound RIO control system (usually the main controller)</description>
+        
+        <channels>
+            <channel id="version" typeId="readonlystring"><label>Firmware Version</label><description>Firmware Version</description></channel>

```

Comment:
Yes, anytime! Whenever you first get hold of the information. If the Thing is stored in a database (and not defined in a *.things file), it will even be persisted.